manifests/02-rbac-role.yaml

```diff
@@ -35,6 +36,7 @@ rules:
   - apps
   resources:
   - deployments
+  - replicasets
```

Comment:
We also don't directly manage `replicasets`.

Reply:
Explained in comment above.